README.md

```diff
@@ -601,7 +601,7 @@ See: [options](#tmp-options)
 
 ```javascript
 cacache.tmp.withTmp(cache, dir => {
-  return fs.writeFileAsync(path.join(dir, 'blablabla'), Buffer#<1234>, ...)
+  return fs.writeFile(path.join(dir, 'blablabla'), Buffer#<1234>, ...)
```

Comment:
Is this right?

Reply:
it started as garbage and i kinda just left it that way.. i'll tweak it to be real code